ci/cases/pr/C96_S2SWA_gefs_replay_ics.yaml

```
@@ -0,0 +1,19 @@
+experiment:
+  system: gefs
+  mode: forecast-only
+
+arguments:
+  pslot: {{ 'pslot' | getenv }}
+  app: S2SWA
+  resdetatmos: 96
+  resdetocean: 1.0
+  resensatmos: 96
+  nens: 2
+  gfs_cyc: 1
+  start: warm
+  comroot: {{ 'RUNTESTS' | getenv }}/COMROOT
+  expdir: {{ 'RUNTESTS' | getenv }}/EXPDIR
+  idate: 2020110100
+  edate: 2020110100
+  yaml: {{ HOMEgfs }}/ci/cases/yamls/gefs_replay_ci.yaml
+  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C96mx100/20240610
```

@NeilBarton-NOAA Should this be `gefs.20201101` instead of `20240610`? 
```suggestion
  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C96mx100/gefs.20201101
```

---

@EricSinsky-NOAA 
on hera
ls /scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96mx100/20240610
gefs.20201031  
gefs.20201101

i.e., the ICs are correct. I don't know about the name of the 20240610 directory.

---

@NeilBarton-NOAA Please disregard my suggested change. I realized `20240610` is a timestamp folder and within this folder is the `20201101` case. It seems this directory exists on Hera, but not the other platforms. @KateFriedman-NOAA When you get a chance, can you sync the IC data under the timestamp folder to the other platforms?

---

Synced to WCOSS2. @WalterKolczynski-NOAA please sync to MSU.

---

Thank you @KateFriedman-NOAA. All looks good in `icsdir` on WCOSS2.

---

MSU is synchronized.